.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,30 @@
+name: Python application
```

Comment:
please remove the whole file. I don't see why we need it here.

Reply:
I need it because my fork is not connected to CI.  I have removed it nevertheless.